core/server/session_logs.go

```diff
@@ -27,6 +28,10 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+const (
+	secretNotCreated = "secret not created"
```

Comment:
⚠️ I think this doesn't need to be a constant as it's only used once, anyway. If we want it to be one let's add a comment explaining what it's used for.

Reply:
@makkes thanks for the comment. We don't know yet if the original issue can be reproduced now or not 100%, I just was not able to reproduce it.

So, this constant is used for consistency with the constant used in a similar issue (which we were able to reproduce): https://github.com/weaveworks/weave-gitops/issues/3297 

If this issue happens again, we'll use a similar constant in the enteprise UI to check specifically for this issue and display a non-error notification to the user.

If we are still unable to reproduce this issue after testing, we can just remove this check completely. But I want to be on the safe side.

Reply:
As discussed, will add this comment and change the return type of the check function to just bool in a next related PR for developer information.